modules/ROOT/pages/table-options.adoc

```diff
@@ -0,0 +1,76 @@
+= Table options
+
+:navtitle: Table options
+:description: TinyMCE tables
+
+== {productname} Table & Advanced Tables plugins
+
+{productname} has built-in table functionality. However, there are two plugins that provide additional functionality for tables:
+
+* Table is an open-source plugin that provides improved table handling through UI components, including dialogs, context menu, context toolbar, menu items and toolbar buttons. For information, see the xref:table.adoc[Table plugin].
+* Advanced Tables is a premium plugin that extends the xref:table.adoc[Table plugin] by adding advanced functionality, including sort options for rows and columns, and row numbering column functionality for tables. For information, see the xref:advtable.adoc[Advanced Tables plugin].
+
+IMPORTANT: The following options apply to the core table functionality and may also affect `table` plugin and `advtable` plugin functionality.
+
+== Controlling how tables are inserted
+
+NOTE: Existing tables and tables added using `+setContent+` or paste are not affected by these options.
```

Comment:
We don't seem to link anywhere else in docs as far as I can see. Maybe we mention them like this?
```suggestion
NOTE: Existing tables and tables added by the `+insertContent+`, `+setContent+` commands/APIs or paste are not affected by these options.
```

Reply:
Not sure what you found, but we generally do... it might be old links that don't? For example: https://github.com/tinymce/tinymce-docs/blob/4949ba2cdf8fc5d289050fed6f7dc5763e886e68/modules/ROOT/partials/configuration/text_patterns.adoc#L12-L13

Reply:
I was more referring specifically to links to the `setContent` and `insertContent` APIs. It makes sense to do that though so will update it

Reply:
Have included the links to the API docs. Thanks for noting that